### PR TITLE
v2ray => 5.3.0

### PR DESCRIPTION
**Pull request by supechicken**
Tested on `x86_64`

### CODE REVIEW

File: packages/v2ray.rb [hunk 1]
```diff
@@ -2,43 +2,40 @@
...
-    puts
-    puts 'You can use customer config. about how to use v2ray command, see https://www.v2ray.com/'.lightblue
-    puts 'If you want to remove v2ray'.lightblue
-    puts
-    puts 'crew remove v2ray'.lightblue
-    puts
+    puts <<~EOT.lightblue
+      To start using v2ray, type `v2ray`.
+
+      You can use customer config. about how to use v2ray command, see www.v2fly.org
```
Comment (satmandu) [inline packages/v2ray.rb:38]: `To use a custom config with the v2ray command, see www.v2fly.org`

**Review by satmandu — CHANGES_REQUESTED** [review]

File: packages/v2ray.rb (hunk 1 above)
Comment (uberhacker) [reply packages/v2ray.rb:38]: See https://www.v2fly.org/en_US/v5/config/overview.html to learn about configuration?

Comment (uberhacker) [reply packages/v2ray.rb:38]: Or for more information, see https://www.v2fly.org/en_US/guide/start.html#novice-guide?

Comment (supechicken) [reply packages/v2ray.rb]: Change made

**Review by satmandu — APPROVED** [review]

**Pull request merged**